### PR TITLE
Updates echo, and makes sure that un-set ARLabOptions don't overwrite the Echo config values

### PR DESCRIPTION
**Pull request by orta**
This should fix the bug that we don't show the video for buy now. 

This PR won't green, I need to update tests which now default to buy now.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Artsy/App/ARAppDelegate+Emission.m [hunk 1]
```diff
@@ -265,6 +254,21 @@ - (void)setupSharedEmissionWithPackagerURL:(NSURL *)packagerURL;
...
+{
+    // Set up all the difference places we get settings to merge into one place
+    NSMutableDictionary *options = [NSMutableDictionary dictionary];
+
+    [options addEntriesFromDictionary:echoFeatures];
+
+    // Handle the fact that it's "AREnableBuyNowFlow" in iOS world - and echo, then "enableBuyNowMakeOffer" in Emission world
+    options[AROptionsBuyNow] = echoFeatures[@"AREnableBuyNowFlow"];
+
+    // Lab options come last (as they are admin/dev controlled, giving them a chance to override)
```
Comment (damassi) [inline Artsy/App/ARAppDelegate+Emission.m]: Always great code comments @orta

**Comment by damassi** [comment]
Thanks for updating 👍Since settings are all merged in I suspect there's no follow up PR in emission?

File: Artsy/App/ARAppDelegate+Emission.m (hunk 1 above)
Comment (orta) [reply Artsy/App/ARAppDelegate+Emission.m]: Thanks, I like to assume that it'll be years later when I see this code again :D

**Pull request merged**